### ISSUE TITLE
TypeError: Cannot read property 'close' of null

### ISSUE THREAD

**Issue by rxin** [langovoi/homebridge-upnp#4]
I ungrouped my Sonos (which I think generated different speaker configurations) and when I restarted homebridge, an exception was thrown:

```
/usr/local/lib/node_modules/homebridge-upnp/node_modules/@langovoi/upnp-device-client/index.js:420
    this.server.close();
                ^
TypeError: Cannot read property 'close' of null
    at DeviceClient.releaseEventingServer (/usr/local/lib/node_modules/homebridge-upnp/node_modules/@langovoi/upnp-device-client/index.js:420:17)
    at MediaRenderer1.onBye (/usr/local/lib/node_modules/homebridge-upnp/MediaRenderer1.js:100:26)
    at SsdpServer._server.on (/usr/local/lib/node_modules/homebridge-upnp/UPnPPlatform.js:92:24)
    at SsdpServer.emit (events.js:188:13)
    at SsdpServer.SSDP._notify (/usr/local/lib/node_modules/homebridge-upnp/node_modules/node-ssdp/lib/index.js:346:12)
    at SsdpServer.parseCommand [as _parseCommand] (/usr/local/lib/node_modules/homebridge-upnp/node_modules/node-ssdp/lib/index.js:311:12)
    at SsdpServer.SSDP._parseMessage (/usr/local/lib/node_modules/homebridge-upnp/node_modules/node-ssdp/lib/index.js:294:10)
    at Socket.onSocketMessage (/usr/local/lib/node_modules/homebridge-upnp/node_modules/node-ssdp/lib/index.js:244:12)
    at Socket.emit (events.js:188:13)
    at UDP.onMessage (dgram.js:685:8)
```

Didn't really impact my usage but you might want to add a check there to make sure "server" is not null. Thanks.

**Comment by langovoi** [comment]
Could you provide full logs (with `-D` argument for Homebridge and `DEBUG=*` env variable) to my email (for your privacy): marklangovoi@gmail.com

**Comment by rxin** [comment]
When I reproduce it I will.

**Comment by langovoi** [comment]
I released 0.1.9 version, which should resolve issue.

**Comment by langovoi** [comment]
I hope new release resolved your issue, if not - re-open.
